formatters/vectorTraffic.js

```diff
@@ -42,6 +51,7 @@ module.exports = function vectorTraffic(fields) {
     fsv.cleanInt(fields.status, 2),
     fields.cacheHit === 'HIT',
     fsv.cleanInt(fields.totalTime, 4),
-    fsv.cleanInt(fields.firstByteTime, 4)
+    fsv.cleanInt(fields.firstByteTime, 4),
+    server
```

Comment:
should have fsv.escapeField(server, 100) just to make sure it matches the schema... obviously its extremely unlikely that it would ever matter, but i've been trying to run everything through an escape function that matches the column its being inserted into.